client/data/speakers.json

```diff
@@ -1,67 +1,226 @@
 {
   "speakers": {
-    "0": { "name": "Una Kravets" },
-    "1": { "name": "Sher Minn Chong" },
-    "2": { "name": "Ziran Sun" },
-    "3": { "name": "TC39 Committee" },
-    "4": { "name": "Tara Z. Manicsic" },
-    "5": { "name": "Stefan Judis" },
-    "6": { "name": "Stanimira Vlaeva" },
-    "7": { "name": "Shwetank Dixit" },
-    "8": { "name": "Pier Paolo Fumagalli" },
-    "9": { "name": "Roshan Gautam" },
-    "10": { "name": "Vitalii Bobrov" },
-    "11": { "name": "Paulo Lopes" },
-    "12": { "name": "Laurie Voss" },
-    "13": { "name": "Jenn Schiffer" },
-    "14": { "name": "Henri Helvetica" },
-    "15": { "name": "C J Silverio" },
-    "16": { "name": "OpenJS Foundation" },
-    "17": { "name": "Michael Mifsud" },
-    "18": { "name": "Manu Martinez-Almeida" },
-    "19": { "name": "Kat Marchán" },
-    "20": { "name": "Jessica Jordan" },
-    "21": { "name": "Jason Williams" },
-    "22": { "name": "Houssein Djirdeh" },
-    "23": { "name": "Garann Means" },
-    "24": { "name": "Bryan Hughes" },
-    "25": { "name": "Chidinma Kalu" },
-    "26": { "name": "Allison McMillan" },
-    "27": { "name": "Alexandra Sunderland" },
-    "28": { "name": "Aditya Mukerjee" },
-    "29": { "name": "Nikhil Thorat" },
-    "30": { "name": "Nick Kreeger" },
-    "31": { "name": "Nat Alison" },
-    "32": { "name": "Meya Stephen Kenigbolo" },
-    "33": { "name": "Maximiliano Firtman" },
-    "34": { "name": "Maxim Koretskyi" },
-    "35": { "name": "max bittker" },
-    "36": { "name": "Martin Sonnenholzer" },
-    "37": { "name": "Marley Rafson" },
-    "38": { "name": "Maël Nison" },
-    "39": { "name": "Leandro Ostera" },
-    "40": { "name": "Kevin Doran" },
-    "41": { "name": "Joyee Cheung" },
-    "42": { "name": "Johnny Austin" },
-    "43": { "name": "Joe Sepi" },
-    "44": { "name": "Jason Straughan" },
-    "45": { "name": "Fedor Indutny" },
-    "46": { "name": "Ella van Durpe" },
-    "47": { "name": "Amanda Sopkin" },
-    "48": { "name": "Constanza Yáñez Calderón" },
-    "49": { "name": "Havi Hoffman" },
-    "50": { "name": "Ashley Williams" },
-    "51": { "name": "Dorian Holmes" },
-    "52": { "name": "Frances Coronel" },
-    "53": { "name": "Hasan Özdemir" },
-    "54": { "name": "Tatiana Mac" },
-    "55": { "name": "Aga Naplocha" },
-    "56": { "name": "Eli Schutze" },
-    "57": { "name": "Emily Rose" },
-    "58": { "name": "Jenn Turner" },
-    "59": { "name": "Manjula Dube" },
-    "60": { "name": "Rachel White" },
-    "61": { "name": "Sareh Heidari" },
-    "62": { "name": "Whitney Williams" }
+    "0": {
```

Comment:
To fix the merge conflicts, merge the original master branch and update your IDs. Please re-order them as well so the numbers are in a logical order.

Also please turn off the formatting.